onesignal-public.php

```diff
@@ -75,20 +75,24 @@ public static function onesignal_header()
             echo 'OneSignal.setDefaultNotificationUrl("'.esc_url($onesignal_wp_settings['default_url']).'");';
         } else {
             echo 'OneSignal.setDefaultNotificationUrl("'.esc_url(get_site_url())."\");\n";
-        } ?>
+        } 
+        
+        if ($onesignal_wp_settings['prompt_auto_register'] === true) {
```

Comment:
These changes call methods directly and do not check for `use_custom_sdk_init` like the `oneSignal_options` was using before. I recommend putting them in the `if (!use_custom_sdk_init)` block, just after OneSignal init

Reply:
RESOLVED